docs/tutorials/avro.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<p><code>for _ in dataset</code>  can be replaced with <code>for record in dataset</code>  for better semantics.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/io/pull/1250/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
thanks, updated.